### ISSUE TITLE
BAUDRATE 1500000 not supported on MacOS development, could you downgrade to B230400?

### ISSUE THREAD

**Issue by triple-it** [friendlyarm/uboot-rockchip#2]
Hi,

We see baudrates of 1500000 appear everywhere...
However MacOS does not support this yet.

CONFIG_SYS_EXTRA_OPTIONS="RKCHIP_RK3399,RKCHIP_RK3399PRO,PRODUCT_MID,NORMAL_WORLD,SECOND_LEVEL_BOOTLOADER,BAUDRATE=1500000"

Screen for example:
~~
tty.c:1090:12: error: use of undeclared identifier 'B1500000'
~~

Maximum supported on MacOSX is B230400, could you please downgrade to 230400 baud?
(There was also some discussion about Posix maximum defined baud rates etc... But 230400 should work out of the box on all development environments)

~~
src % grep -B3 B230400 /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/sys/termios.h
#define B57600  57600
#define B76800  76800
#define B115200 115200
#define B230400 230400
src % 
~~

**Comment by axxyhtrx** [comment]
I fully agree here, no need to install driver for macos in this case. It will work out of the box without additional software. Make it simple, please.
